python/cudf_polars/cudf_polars/experimental/benchmarks/utils_new_frontends.py

```
@@ -437,6 +440,12 @@ def __post_init__(self) -> None:  # noqa: D105
                 "iteration 0 warms the cache, iterations 1+ are the hot measurements."
             )
 
+        # Update `extra_info.environment` with the captured environment variables.
+        self.extra_info.setdefault("environment", {})
+        for var in self.capture_env_vars.split(","):
+            var = var.strip()
```

Ruff (0.15.12)

[warning] `for` loop variable `var` overwritten by assignment target

(PLW2901)